workspace_hack/Cargo.toml

```diff
@@ -36,12 +38,16 @@ prost = { version = "0.11" }
 rand = { version = "0.8", features = ["small_rng"] }
 regex = { version = "1" }
 regex-syntax = { version = "0.6" }
+reqwest = { version = "0.11", default-features = false, features = ["blocking", "json", "rustls-tls"] }
```

Comment:
surprised to see the blocking feature here. Even though it's only a bit more methods enabled, I cannot immediatedly see why it's being enabled.